src/containers/ListingPage/ListingPage.js

```diff
@@ -423,26 +418,12 @@ export class ListingPageComponent extends Component {
                   />
                   <SectionDescriptionMaybe description={description} />
                   <SectionFeaturesMaybe options={yogaStylesConfig} publicData={publicData} />
-                  <SectionRulesMaybe publicData={publicData} />
                   <SectionMapMaybe
                     geolocation={geolocation}
                     publicData={publicData}
                     listingId={currentListing.id}
                   />
                   <SectionReviews reviews={reviews} fetchReviewsError={fetchReviewsError} />
-                  <SectionHostMaybe
-                    title={title}
-                    listing={currentListing}
-                    authorDisplayName={authorDisplayName}
-                    onContactUser={this.onContactUser}
-                    isEnquiryModalOpen={isAuthenticated && this.state.enquiryModalOpen}
```

Comment:
If you remove `SectionHostMaybe` component, is there some other way to open enquiry modal? SectionHeading needs to be able to open this modal - so my guess is that you need to move Modal from SectionHost to this file. (and don't delete props `sendEnquiryInProgress`, `sendEnquiryError`).